pkg/cmd/server/bootstrappolicy/project_policy.go

```diff
@@ -42,3 +42,29 @@ func GetBootstrapServiceAccountProjectRoleBindings(namespace string) []authoriza
 		},
 	}
 }
+
+func GetBootstrapTemplateServiceBrokerProjectRoleBindings(namespace string) []authorizationapi.RoleBinding {
+	return []authorizationapi.RoleBinding{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      TemplateServiceBrokerRoleBindingName,
+				Namespace: namespace,
+			},
+			RoleRef: kapi.ObjectReference{
+				Name: TemplateServiceBrokerRoleName,
+			},
+			Subjects: []kapi.ObjectReference{
+				{
+					Kind:      authorizationapi.ServiceAccountKind,
+					Name:      InfraTemplateServiceBrokerServiceAccountName,
+					Namespace: DefaultOpenShiftInfraNamespace,
+				},
+			},
+		},
+	}
+}
+
+func GetBootstrapProjectRoleBindings(namespace string) []authorizationapi.RoleBinding {
+	return append(GetBootstrapServiceAccountProjectRoleBindings(namespace),
+		GetBootstrapTemplateServiceBrokerProjectRoleBindings(namespace)...)
```

Comment:
Granting rights for an alpha feature?  Can you separate this particular bit to a different pull?  I'd expect it to be gated based on whether the feature is enabled or not, but I don't want to block progress on the rest of your pull.